Sources/Plot/API/HTMLElements.swift

```diff
@@ -247,6 +247,12 @@ public extension Node where Context: HTML.BodyContext {
         .element(named: "i", nodes: nodes)
     }
 
+    /// Add an `<!--[if IE]>...<![endif]-->` HTML element within the current context.
+    /// - parameter nodes: The element's attributes and child elements.
+    static func ifIE(_ nodes: Node<Context>...) -> Node {
```

Comment:
Not sure if I think this is worth including in Plot itself, as it can be achieved using an HTML comment. Also, do people still use this in practice? I thought we were past the days of Internet Explorer-special casing?

Reply:
After some thought, I do have to agree, but for different rationale.  I added it because my attempts to add [Conditional Comments](https://en.wikipedia.org/wiki/Conditional_comment) inside .comment() failed to be parsed properly by at least Firefox.  The effort required to fully implement conditional comments within the DSL would be more than this commit, and likely not worth the effort for this limited edge case since it can still be implemented with .raw() so I removed the commits.

As to your questions, it is still used in practice, but the use cases are more limited for sure.  Many users still do use EOL old versions of IE for some reason.